src/stcal/alignment/util.py

```diff
@@ -227,13 +227,18 @@ def _calculate_offsets(fiducial: tuple,
         A two-elements array containing the minimum pixel value for each axis.
 
     crpix : list or tuple
-        Pixel coordinates of the reference pixel.
+        0-indexed pixel coordinates of the reference pixel.
 
     Returns
     -------
     ~astropy.modeling.Model
         A model with the offsets to be added to the WCS's transform.
 
+    tuple
+        A tuple of offset *values* that are to be *subtracted* from input
+        coordinates (negative values of offsets in the returned transform).
+        Note: these are equivalent to an effective 0-indexed "crpix".
```

Comment:
Does this mean that these offsets are always equal to crpix?  If so, what is the point of returning this, if crpix is an input to the function?  If not, can you clarify this note a little bit?